packages/itwin/tree-widget/src/tree-widget-react/components/trees/common/components/VisibilityTree.tsx

```diff
@@ -6,9 +6,8 @@
 import { useCallback } from "react";
 import { useIModelAccess } from "../internal/UseIModelAccess.js";
 import { useHierarchyVisibility } from "../UseHierarchyVisibility.js";
-import { TreeBase } from "./Tree.js";
+import { Tree, TreeProps } from "./Tree.js";
```

Comment:
Why did TreeProps import changed?

Reply:
did not notice this, weird it should not have consolidated them. Reverted